### ISSUE TITLE
Travis errors

### ISSUE THREAD

**Issue by bgyori** [sorgerlab/indra#338]
Some of the latest from Travis. This happens right after the call to `nosetests`
```
RuntimeError: module compiled against API version 0xb but this version of numpy is 0xa
```
This one happens a lot during tests.
```
Failure: ImportError (C extension: iNaT not built. If you want to import pandas from the source directory, you may need to run 'python setup.py build_ext --inplace --force' to build the C extensions first.) ... ERROR
Failure: ImportError (cannot import name PysbAssembler) ... ERROR
Failure: ImportError (cannot import name CxAssembler) ... ERROR
```
Probably something to do with the `numpy` install on Travis.

**Comment by pagreene** [comment]
Actually, I think it may be an issue with the Pandas install. It looks to me like Pandas (or maybe something else) was compiled with numpy 0xb, but numpy is 0xa, which apparently is not ok.

**Comment by bgyori** [comment]
Fixed in #339
